e2etests/bats-tests.sh

```diff
@@ -0,0 +1,527 @@
+#!/usr/bin/env bats
+
+load bats-support-clone
+load test_helper/bats-support/load
+load test_helper/redhatcop-bats-library/src/error-handling.bash
+
+@test "access-to-create-pods" {
+  tmp="tests/checks/access-to-create-pods.yml"
+  cmd="${KUBE_LINTER_BIN} lint --include access-to-create-pods --do-not-auto-add-defaults ${tmp}"
+  run ${cmd}
+
+  print_info "${status}" "${output}" "${cmd}" "${tmp}"
+  [ "$status" -eq 1 ]
+  [[ "${lines[1]}" == "tests/checks/access-to-create-pods.yml: (object: namespace-dev/role-binding1 rbac.authorization.k8s.io/v1, Kind=RoleBinding) binding to \"role1\" role that has [create] access to [pods] (check: access-to-create-pods, remediation: Where possible, remove create access to pod objects in the cluster.)" ]]
+  [[ "${lines[2]}" == "Error: found 1 lint errors" ]]
+  [[ "${#lines[@]}" -eq 3 ]]
+}
```

Comment:
There is a lot of boilerplate here. I wonder if it would be possible to specify this more declaratively somehow, versus requiring all this to be copy pasted?

Reply:
I can't think of a less copy+paste way than what I've currently done, but open to suggestions/ideas

Reply:
Fair enough. Looking at it again, boilerplate is quite minimal.